api/lib/application/certification-centers/certification-center-controller.js

```diff
@@ -0,0 +1,45 @@
+const usecases = require('../../domain/usecases');
+const certificationCenterSerializer = require('../../infrastructure/serializers/jsonapi/certification-center-serializer');
+const controllerReplies = require('../../infrastructure/controller-replies');
+const {
+  NotFoundError: InfrastructureNotFoundError,
+} = require('../../infrastructure/errors');
+
+const {
+  NotFoundError,
+} = require('../../domain/errors');
+
+module.exports = {
+
+  save(request, h) {
+    const certificationCenter = certificationCenterSerializer.deserialize(request.payload);
+    return usecases.saveCertificationCenter({ certificationCenter })
+      .then((certificationCenter) => certificationCenterSerializer.serialize(certificationCenter))
```

Comment:
On peut simplifier ici en :
.then(certificationCenterSerializer.serialize)

Reply:
oui

Reply:
pas sûr (s’il y a un `this` quelque part dans la méthode, c’est mort)

Reply:
ici à mon avis on risque rien vu qu'on fait des `new Serializer` dedans donc le context est réinitialisé à un objet vide. Mais par contre pour le hapi tu as raison, c'est risqué, on ne fait pas.